### ISSUE TITLE
I want to continue this project

### ISSUE THREAD

**Issue by yfchina143** [ohyou/twitch-viewer#47]
Hi i want to continue this project, but i am not sure where to start. So far i made it working by replacing some of the old stuff such as livestreamer to some other software and () all the prints. However, think isn't enough to make it work. I was wondering what is the client_id you have on line 54. and what should i look into to make this work again

**Comment by direct-fuel-injection** [comment]
@yfchina143 hi, do you have any success to make it work?

**Comment by yfchina143** [comment]
Yes, in a different way , which isn't efficient. I used selenium and
special ips

**Comment by selfpromochat** [comment]
If you guys get it to work plz reply

**Comment by h1rule** [comment]
Thumb up @yfchina143 does it work?

**Comment by pythonnoob2** [comment]
Any updates? @yfchina143

**Comment by yfchina143** [comment]
yeah, so instead of using this, i just made my own. with selenium. its kind of dumb and it eats up alot of resource. not worth it i would say. could probably get you like 10 viewer+ depands on ur computer. i say the best easy way is to buy it. 40$ -30$ a month could get u enough viewer to start. just google python viewer bot. if you do want to try my let me know , ill create a github repo for yall

**Comment by pythonnoob2** [comment]
I have a pretty powerful 2nd computer laying around.  I would like to give it a shot, if you don't mind? @yfchina143 

edit: plus i am sure others would like to try it as well

**Comment by pythonnoob2** [comment]
I started writing one in selenium.  I have it half working.  It opens up the new windows in firefox, but it's not reading my proxy list.  It still show's my ip on the new windows it is opening, so no new views are being sent to the channel.  Any thoughts? @yfchina143

**Comment by yfchina143** [comment]
How are u getting the proxy?

**Comment by pythonnoob2** [comment]
like within the code or which proxies am I using?  just using free http proxies at the moment. 

edit: this is the code I am using. The proxies are stored in the file.txt

https://pastebin.com/izv2Lves

**Comment by yfchina143** [comment]
It's the free proxy. They don't work. You need to buy Private proxy. Think
about how easy twitch can get them too and have them in a blocked list. I
did the same thing u did. I end up buying private ips

**Comment by pythonnoob2** [comment]
Do you have a recommended site to buy them from? 

edit: so the code looks good, its just the proxies?

**Comment by yfchina143** [comment]
I forgot, but look for eilte ips or something like that

**Comment by pythonnoob2** [comment]
Any chance you want to share your code so I can compare yours to mine?

**Comment by yfchina143** [comment]
Sure , give me a few days, been busy with work. The code is in my old
laptop, need to dig for it

**Comment by pythonnoob2** [comment]
Alright, dope! Thank you

**Comment by freetextmike** [comment]
@yfchina143 can you post your code on your profile or if not , can you share it with me too i have the resources and can definitely buy premium proxies

**Comment by yfchina143** [comment]
https://github.com/yfchina143/twitchViewerBot
sorry about the delay, been busy with work lately. this is what i can find. pretty easy script only a few lines.

**Comment by yfchina143** [comment]
let me know if u guys have any question.

**Comment by pythonnoob2** [comment]
Thanks for uploading! I am going to test out shortly.  I see you have chromedriver.exe in your repo.  I was planning on using firefox, if so.  Do i still need the chromedriver? i have gecko and FF working together now with my current code.

**Comment by rickybarbour** [comment]
@yfchina143 do you know if this works with firefox and chromedriver, or just one? i noticed you had both drivers in the repo and in the code from what it looked like

**Comment by yfchina143** [comment]
Both will work,  just got to download different driver for it, I was trying
to do headless for different driver to reduce the resources, but you cannot
do headless, it won't count as a view.

**Comment by yfchina143** [comment]
> Thanks for uploading! I am going to test out shortly. I see you have chromedriver.exe in your repo. I was planning on using firefox, if so. Do i still need the chromedriver? i have gecko and FF working together now with my current code.

srry about the late reply, python, drivers are just options you can have. doesn't matter which driver. i was testing it because i wanted to see how each driver are different and to see if i can do headless. which can save tons of resource. but failed

**Comment by unspike** [comment]
@yfchina143, @rickybarbour, @pythonnoob2, @freetextmike 
Have you guys managed it to work? Can I have a few lines of how to use it?

**Comment by brunomaffei** [comment]
How can i use this code? Make it works?

**Comment by Lejobuild** [comment]
@yfchina143 is ur link a youtube view bot? there is a line with a youtube link

**Comment by yfchina143** [comment]
> @yfchina143 is ur link a youtube view bot? there is a line with a youtube link

its for twitch.

**Comment by yfchina143** [comment]
> How can i use this code? Make it works?

u need to pay for elite ip address. then put it in one of the files. lastly, read the IP from the files. it will open up browsers with your stream. but it browsers will take up rams. 8gb of ram could probably do 4-7 browsers. if u do more, your computer will lag up. i dont recommend doing it my way.
